cmd/packageImport.go

```diff
@@ -240,6 +246,31 @@ func importPackagesFromCollection(status *statuswriter.Sink, url string, exclude
 	status.Write(message.Ssuccessf("successfully imported %d of %d package from collection %s", successfulImports, len(pkgs), parsedURL.String()))
 }
 
+func importPackageFromStdin() {
+	sw := statuswriter.New()
+	scanner := bufio.NewScanner(os.Stdin)
+	input := ""
+	for scanner.Scan() {
+		input += scanner.Text() + "\n"
+	}
+	sw.Run()
+	status := sw.NewSink()
+	defer sw.Wait()
+	defer status.Close()
+	pkg, err := session.packageService.ImportPackageFromString(input)
+	if err != nil {
+		status.Write(message.Serrorf(err, "failed to import package from standard input"))
+		return
+	}
+	err = session.packageService.StorePackage(pkg)
+	if err != nil {
+		status.Write(message.Serrorf(err, "failed to store package"))
+		return
+	}
+
+	status.Write(message.Ssuccessf("successfully imported package %s [%s]", pkg.Name, pkg.Label))
+}
+
```

Comment:
Again, good job! I'm just posting the updated function below to safe you some work in case you agree on my proposed changes from above. 

```go
func importPackageFromStdin(status *statuswriter.Sink) (*domain.Package, error) {
	// Scan package config from stdin into string
	status.Write(message.Sinfof("importing package from stdin"))
	scanner := bufio.NewScanner(os.Stdin)
	input := ""
	for scanner.Scan() {
		input += scanner.Text() + "\n"
	}

	// Import actual package from the string
	pkg, err := session.packageService.ImportPackageFromString(input)
	if err != nil {
		return nil, err
	}

	// Save the package
	status.Write(message.Sinfof("storing package %s [%s]", pkg.Name, pkg.Label))
	err = session.packageService.StorePackage(pkg)

	return pkg, err
}
```